parser/field.go

```diff
@@ -231,6 +231,11 @@ func (p *Parser) parseGoProtoValidatorFieldOptionConstant() (string, error) {
 		switch {
 		case p.lex.Token == scanner.TCOMMA:
 			ret += p.lex.Text
```

Comment:
I am unsure whether this needs to be here, or after the if statement.
Moving it after changes the output in tests from `{field: data,}` to `{field: data}`; is the parser expected to remove this kind of syntactic sugar? or is it remaining non-opinionated?

Reply:
No problem!
I fully agree with being non-opinionated; I see no reason to remove it.